Server/mods/deathmatch/logic/packets/CPlayerPuresyncPacket.cpp

```diff
@@ -77,13 +77,30 @@ bool CPlayerPuresyncPacket::Read(NetBitStreamInterface& BitStream)
         if (!BitStream.Read(&position))
             return false;
 
-        if (pContactElement != nullptr &&
-            (!IsPointNearPoint3D(pSourcePlayer->GetPosition(), pContactElement->GetPosition(), g_TickRateSettings.iVehicleContactSyncRadius) ||
-                pSourcePlayer->GetDimension() != pContactElement->GetDimension()))
+        if (pContactElement != nullptr)
         {
-            pContactElement = nullptr;
-            // Use current player position. They are not reporting their absolute position so we have to disregard it.
-            position.data.vecPosition = pSourcePlayer->GetPosition();
+            float radius = -1;
```

Comment:
It's better to use int32_t here because iVehicleContactSyncRadius and iObjectContactSyncRadius have integer types. In addition comparison of integers (line 96) is simpler and more reliable.

Reply:
Done